query-node/mappings/src/council.ts

```diff
@@ -433,11 +436,15 @@ export async function council_NewCouncilElected({ event, store }: EventContext &
   // get election round and its candidates
   const electionRound = await getCurrentElectionRound(store)
 
-  // TODO: uncomment when following query will be working (after some QN patches make it to Olympia)
-  // const electedCandidates = await store.getMany(Candidate, { where: { electionRoundId: electionRound.id, member: { id_in: electedMemberIds } } })
-  const electedCandidates = (
-    await store.getMany(Candidate, { where: { electionRoundId: electionRound.id }, relations: ['member'] })
-  ).filter((item: Candidate) => electedMemberIds.find((tmpId) => tmpId === item.member.id.toString()))
+  const where = { electionRoundId: electionRound.id, status_json: { isTypeOf_eq: 'CandidacyStatusActive' } }
```

Comment:
This doesn't look like a valid TypeORM where condition.
Note that we're not querying the graphql server here, but the PostgreSQL database instead.
To query by `status` you'd need a condidion like:
```suggestion
  const where = {
    electionRoundId: electionRound.id,
    status: Raw((alias) => `${alias}::json->'isTypeOf' = :status`, {
      status: 'CandidacyStatusActive',
    })
  }
```
Where `Raw` is:
```typescript
import { Raw } from 'typeorm'
```

Reply:
Sorry about that I'm not familiar with TypeORM, I wrote that as a placeholder and forgot about it :facepalm: